engines/private/module.mk

```diff
@@ -0,0 +1,25 @@
+MODULE := engines/private
+ 
+MODULE_OBJS := \
+	metaengine.o \
```

Comment:
It is best to sort this list alphabetically

Reply:
Done.